core-services/prow/02_config/_config.yaml

```diff
@@ -710,6 +712,25 @@ tide:
     - do-not-merge/hold
     - do-not-merge/work-in-progress
     - do-not-merge/invalid-owners-file
+    repos:
+    - openshift-priv/openshift-tests
+    - openshift-priv/openshift-tests-private
+    - openshift/openshift-tests-private
+    - openshift/openshift-tests
+  - includedBranches:
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - master
+    labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
```

Comment:
This is close, but incorrect. For some reason you changed the configuration for the `ocs-operator` repos, which you shoudn't (note the differnt indent before the `includedBranches`.

You need to add a new list item (representing a GH query) that will set 1) labels that must be present (lgtm + approved) 2) labels that must be present (needs-rebase, the three do-not-merge one and the bugzilla/invalid-bug), the list of repos where this applies, and eiter the list of branches that should be included, or the list of branches taht should be excluded (implying that all others are inclded).

You should not modify any of the existing queries, just add a new one.

Reply:
Sorry for that, my mistake. Updated it.